superset-frontend/stylesheets/antd/index.less

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+@import '~antd/lib/style/themes/index';
+@import '~antd/lib/style/mixins/index';
+@import '~antd/lib/style/core/base';
+
+*[class*='ant-'] {
+  @import '~antd/lib/style/core/global.less';
+}
+
+@import '~antd/lib/style/core/iconfont';
+@import '~antd/lib/style/core/motion';
+@import '~antd/lib/style/components.less';
+
+/*
+  Theme variables here: https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less
+*/
+@primary-color: #20a7c9;
```

Comment:
These colors should be pulled from `variables.less` rather than being hard-coded here.

Reply:
Or better yet, we could pull these variables in via JS using something like [this](https://github.com/mzohaibqc/react-app-rewire-antd-theme#readme) or (perhaps less ideal, but easier) [this](https://webpack.js.org/loaders/less-loader/#prependdata).

Reply:
Yeah, finding a single-source solution to this will make a big difference for custom themes, or switching light and dark mode. I think ideally these variables would be set at runtime, not at build time.

Reply:
referencing `variables.less` is a good first step

Reply:
I'm a little concerned with including that file since we might unintentionally overwrite variables we don't mean to. If we want to do this right, there should be a root file that just namespaces the constants.

E.g.
```
// In constants.less
@superset-color-primary = #20a7c9;

// In variables.less
@primary-color = @superset-color-primary;

// In antd.less
@primary-color = @superset-color-primary;
```

Reply:
`variables.less` is kind of bloated right now and it's difficult to distinguish between our custom variables and the Bootstrap/Cosmos ones. Adding in ant variables will creating a even bigger mess.

I think a larger question is do we want to keep `less` in our stack or do we want to slowly migrate away from it? The larger frontend community seems to have been [favoring](https://nextjs.org/docs/advanced-features/customizing-postcss-config) PostCSS, CSS variables, and CSS-in-JS recently. If we also believe these technologies are the future, we should probably limit our use of less to creating a base theme at build-time, and add runtime theme overrides using only styled components or css variables.